feat.: Azure Docs (#20706)
newrelic/docs-website: src/content/docs/serverless-function-monitoring/azure-function-monitoring/compatibility-requirement-azure-monitoring.mdx

<Tabs>
  <TabsBar>

    <TabsBarItem id="linux">Linux</TabsBarItem>
    <TabsBarItem id="windows">Windows</TabsBarItem>
    <TabsBarItem id="container">Containerized functions</TabsBarItem>

  </TabsBar>
  <TabsPages>
  

<TabsPageItem id="linux">

<table>
  <thead>
    <tr>
      <th>Runtime</th>
      <th>Requirements</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Node.js</td>
      <td>
        * New Relic NodeJS agent version 12.18.0 or above.
        * NodeJS version 18 or above.
      </td>
    </tr>
    <tr>
      <td>Python</td>
      <td>

>         * New Relic Python agent version `information needed` or above.

Reviewer: ```suggestion
        * New Relic Python agent version 10.13.0 or above.
```

    <tr>
      <td>.NET</td>
      <td>
        * New Relic .NET agent version 10.37.0 or above.
        * .NET stack: 
          * .NET 6 - 9, Isolated model only
      </td>
    </tr> 
  </tbody>
</table>

</TabsPageItem>

<TabsPageItem id="windows">

<table>
  <thead>
    <tr>
      <th>Runtime</th>
      <th>Requirements</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Node.js</td>
      <td>
        * New Relic NodeJS agent version 12.18.0 or above.
        * NodeJS version 18 or above.
      </td>
    </tr>
    <tr>
      <td>.NET</td>
      <td>
        * New Relic .NET agent version 10.37.0 or above.
        * .NET stack: 
          * .NET 4.8
          * .NET 6 - 9, Isolated model only
          * .NET 6 and 8, In-process model (.NET agent version 10.38.0 and later)
      </td>
    </tr> 
  </tbody>
</table>

</TabsPageItem>

<TabsPageItem id="container">

<table>
  <thead>
    <tr>
      <th>Runtime</th>
      <th>Requirements</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Node.js</td>
      <td>
        * New Relic NodeJS agent version 12.18.0 or above.
        * NodeJS version 18 or above.
      </td>
    </tr>
    <tr>
      <td>Python</td>
      <td>

>         * New Relic Python agent version `information needed` or above.

Reviewer: ```suggestion
        * New Relic Python agent version 10.13.0 or above.
```

## Supported triggers and Distributed Tracing

<CollapserGroup>

  <Collapser
    id="node"
    title="For NodeJS"
  >

  You can instrument the following trigger type:

  * HttpTrigger 
  * HTTPTrigger for response streaming
  * TimerTrigger

  </Collapser>

  <Collapser
    id="python"
    title="For Python"
  >

  You can instrument the following trigger type:

  * HttpTrigger 

  </Collapser>

  <Collapser
    id=".net"
    title="For .NET"
  >

>   You can instrument all trigger types, however Distributed Tracing is supported only in the following cases:

Reviewer 2: ```suggestion
  You can instrument all trigger types, except Distributed Tracing which is supported only in the following cases:
```
